### ISSUE TITLE
[V4 Images] Core tools docker build fails with - The repository 'https://packages.microsoft.com/repos/azure-cli bullseye Release' does not have a Release file.

### ISSUE THREAD

**Issue by balag0** [Azure/azure-functions-docker#474]
\host\4\bullseye\amd64\dotnet\dotnet-inproc\dotnet-core-tools.Dockerfile
and
\host\4\bullseye\amd64\dotnet\dotnet-isolated\dotnet-isolated-core-tools.Dockerfile

do not build currently. 

Error:

	&& echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/azure-cli.list \
	&& apt-get update \
	

Fails with
	
Get:7 https://packages.microsoft.com/debian/9/prod stretch/main amd64 Packages [155 kB]
#5 24.19 Reading package lists...
#5 24.72 E: The repository 'https://packages.microsoft.com/repos/azure-cli bullseye Release' does not have a Release file.

+@AnatoliB for someone to help with coretools.

**Comment by pragnagopa** [comment]
@TsuyoshiUshio - is this the same issue as fix - https://github.com/Azure/azure-functions-docker/commit/4741d3b28e475f0c26514b5826420759b33bc7d0 ?
If yes, can you please help here?

**Comment by TsuyoshiUshio** [comment]
hi @balag0 
The same. Currently, Azure CLI team working on it. Bullseye is not supported for AzureCLI. For patching we need to fix the version. like this. https://github.com/Azure/azure-functions-docker/pull/489

My case was python image (that is internally used on the Dockerfile) os has move from buster to bullseye. For analyze it, I need more messages. 

Probably we should consider to change like this. 
https://github.com/Azure/azure-functions-docker/issues/494

We can see the other OS, however, the directory name is bullseye, so that better to wait the Azure CLI is being ready for bullseye. 
https://mcr.microsoft.com/v2/dotnet/sdk/tags/list

This represents the status of the Bullseye release. Once it has done, It should be fine. 
https://github.com/Azure/azure-cli/pull/19370

**Comment by balag0** [comment]
yeah agree. I think waiting for the bullseye release for core tools should be fine since this image is not needed right away.
+@anthonychu for inputs as well.

**Comment by vivekjilla** [comment]
This build failure was resolved at the time via PR #489 once the Azure CLI packaging caught up. The current images no longer carry that workaround. Closing as resolved.
